### DISCUSSION TITLE
[委托]物竞复赛

### DISCUSSION THREAD

**Discussion by Anslate** [share121/inter-knot#5228] [委托]
![IMG_20240920_103556](https://github.com/user-attachments/assets/98f5bcd5-2131-4343-99dd-22c85da18c18)
就是前几周被老师骗过去考了个预赛。谁知今年预赛这么简单，压线进了复赛。可是我现在啥也不会。🤯
现在准备去南师大附中考试了，有没有谁能把这变成空洞，让考试延期啊。
有意者私聊～

**Comment by ParanoidHikari** [comment]
绳匠里有能手搓空洞的人物吗??

**Comment by Kazuha-hhh** [comment]
这个委托我接下了

**Comment by Kazuha-hhh** [comment]
楼主先去测个以太适性
